docs/cryoet_data_portal_docsite_FAQ.md

```diff
@@ -1,4 +1,6 @@
 # FAQ
 **To be updated after release**
+.. collapse:: Details
 
+    Example Text to see if the extension worked properly.
```

Comment:
I believe this might need to updated. :)

Reply:
Whoops, I didn't realize adding more commits after opening the PR would incorporate as well.